tools-local/targetgroups.props

```diff
@@ -189,33 +189,46 @@
       <Imports>netcoreapp2.0</Imports>
       <CompatibleWith>netstandard</CompatibleWith>
     </TargetGroups>
-    <!-- netcoreapp is an alias for netcoreapp2.1 -->
-    <TargetGroups Include="netcoreapp">
-      <TargetFramework>netcoreapp2.1</TargetFramework>
-      <NuGetTargetMoniker>.NETCoreApp,Version=v2.1</NuGetTargetMoniker>
-      <NuGetTargetMonikerShort>netcoreapp2.1</NuGetTargetMonikerShort>
+    <TargetGroups Include="netcoreapp2.2">
+      <TargetFramework>netcoreapp2.2</TargetFramework>
+      <NuGetTargetMoniker>.NETCoreApp,Version=v2.2</NuGetTargetMoniker>
+      <NuGetTargetMonikerShort>netcoreapp2.2</NuGetTargetMonikerShort>
       <Imports>netcoreapp2.1</Imports>
       <CompatibleWith>netstandard</CompatibleWith>
     </TargetGroups>
-    <TargetGroups Include="netcoreapp1.2">
-      <TargetFramework>netcoreapp1.2</TargetFramework>
-      <NuGetTargetMoniker>.NETCoreApp,Version=v1.2</NuGetTargetMoniker>
-      <NuGetTargetMonikerShort>netcoreapp1.2</NuGetTargetMonikerShort>
+    <TargetGroups Include="netcoreapp3.0">
+      <NETCoreAppMaximumVersion>3.0</NETCoreAppMaximumVersion>
```

Comment:
Question: What are you using this for?

Reply:
This is needed to build our projects now given that the CLI we are using doesn't support this 3.0 as a target framework so we need to set the max version to be able to build.